packages/core/src/config/sources.ts

```diff
@@ -1,16 +1,23 @@
 import type { Abi, Address, Hex, LogTopic } from "viem";
 import type { AbiEvents } from "./abi.js";
 
+type Topic0 = Hex | Hex[];
+export type Topics =
```

Comment:
This seems fine. Might be more accurate to name this `TopicFilter`, actually. IMO `Topics` more accurately describes the `RpcLog["topics"]` type.